src/test/ui/iterators/collect-into-slice.rs

```diff
@@ -1,15 +1,20 @@
 fn process_slice(data: &[i32]) {
     //~^ NOTE required by a bound in this
+    //~| NOTE required by a bound in this
     todo!()
 }
 
 fn main() {
     let some_generated_vec = (0..10).collect();
     //~^ ERROR the size for values of type `[i32]` cannot be known at compilation time
+    //~| ERROR the size for values of type `[i32]` cannot be known at compilation time
```

Comment:
Does this one dedup in practice?

Reply:
No :/ I don't actually know why these deduplicated before. One is due to the implicit sized on the generic `<B>` in `Iterator::collect`, and then one's a derived impl.